images/dnsmasq/Makefile

```diff
@@ -27,9 +27,9 @@ OUTPUT_DIR := _output/$(ARCH)
 # Ensure that the docker command line supports the manifest images
 export DOCKER_CLI_EXPERIMENTAL=enabled
 
-BASEIMAGE ?= k8s.gcr.io/debian-base-$(ARCH):v2.0.0
+BASEIMAGE ?= us.gcr.io/k8s-artifacts-prod/build-image/debian-base-$(ARCH):v2.1.0
```

Comment:
Is this because k8s.gcr.io still points to google_containers? Can this be used for building from any location?

Reply:
yeah im not sure what caused them to change the location. The us.gcr.io domain is used now for all k8s builds so Im guessing its reliable from all locations.

Reply:
@listx Could you confirm this is ok to use? 

Once this is resolved, i think this is ok to merge.

Reply:
@prameshj -- Yep. This is the US endpoint for community-hosted images now. See this tracking issue: https://github.com/kubernetes/kubernetes/issues/58012

The VDF (vanity domain flip for k8s.gcr.io) is being tracked here: https://github.com/kubernetes/release/issues/270

The other usable endpoints are asia.gcr.io and eu.gcr.io.

Reply:
thanks. I see https://github.com/kubernetes/kubernetes/pull/90697/files uses the same us endpoint.